{"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError\/serverError"},"kind":"symbol","sections":[],"schemaVersion":{"major":0,"minor":3,"patch":0},"hierarchy":{"paths":[["doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id","doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError"]]},"metadata":{"symbolKind":"case","title":"IdVerificationError.serverError","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"serverError","kind":"identifier"}],"role":"symbol","modules":[{"name":"IdVerification365id"}],"roleHeading":"Case","externalID":"c:@M@IdVerification365id@E@IdVerificationError@IdVerificationErrorServerError"},"abstract":[{"text":"Happens when there is an error thrown by the 365id server","type":"text"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/idverification365id\/idverificationerror\/servererror"]}],"primaryContentSections":[{"declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"serverError","kind":"identifier"}],"platforms":["iOS"],"languages":["swift"]}],"kind":"declarations"}],"references":{"doc://com.365id.sdk.idverification/documentation/IdVerification365id":{"abstract":[],"kind":"symbol","title":"IdVerification365id","role":"collection","type":"topic","identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id","url":"\/documentation\/idverification365id"},"doc://com.365id.sdk.idverification/documentation/IdVerification365id/IdVerificationError":{"url":"\/documentation\/idverification365id\/idverificationerror","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"IdVerificationError","kind":"identifier"}],"title":"IdVerificationError","type":"topic","kind":"symbol","role":"symbol","identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError","navigatorTitle":[{"text":"IdVerificationError","kind":"identifier"}],"abstract":[{"type":"text","text":"Id Verification Errors used in the "},{"type":"codeVoice","code":"onError(error:)"},{"type":"text","text":" method in the "},{"type":"codeVoice","code":"IdVerificationEventDelegate"}]},"doc://com.365id.sdk.idverification/documentation/IdVerification365id/IdVerificationError/serverError":{"abstract":[{"type":"text","text":"Happens when there is an error thrown by the 365id server"}],"kind":"symbol","url":"\/documentation\/idverification365id\/idverificationerror\/servererror","title":"IdVerificationError.serverError","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"serverError"}],"role":"symbol","identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError\/serverError","type":"topic"}}}